_episodes/04-sharing-environments.md

```diff
@@ -314,18 +340,18 @@ The `--prune` option tells Conda to remove any dependencies that are no longer r
 >  - matplotlib=3.1
 >  - pandas=0.24
 >  - scikit-learn=0.21
->  - pip=19.1
+>  - pip=22.3

```

Comment:
Maybe Python instead?

Reply:
Ah, if `pip` is bundled with Python by default there are quite a few places that need changing to remove explicit mention of `pip` and instead just reference `python`. Back in Episode 3 there is an explanation about avoiding using the system or Conda base `pip`.

On my branch under episode 3 line 364/365 (where lines have been broken so numbers don't align, its the config I've got setup on Emacs and I learnt that Markdown renders them as continuous lines anyway).

> If you find yourself needing to install a Python package that is only available via PyPI, then you should first install
`pip` into your Conda environment and then use that `pip` to install the desired package from PyPI.

Then under the solution `pip` is explicitly installed...

```
> > $ conda install name machine-learning-env pip
```

So there is a bit more work to be done on this front, but as with the above issue I'll save that for another round of reviews and get our current PRs merged.